src/converters.ts

```diff
@@ -1,5 +1,20 @@
-const unconvertable = (): undefined => undefined;
+import { Codec } from './components/types';
+import ArrayType from './components/types/Array';
+import BooleanType from './components/types/Boolean';
+import ButtonType from './components/types/Button';
+import CheckboxType from './components/types/Checkboxes';
+import ColorType from './components/types/Color';
+import DateType from './components/types/Date';
+import FilesType from './components/types/Files';
+import NumberType from './components/types/Number';
+import ObjectType from './components/types/Object';
+import OptionsType from './components/types/Options';
+import RadioType from './components/types/Radio';
+import SelectType from './components/types/Select';
+import TextType from './components/types/Text';
+import { KnobType } from './type-defs';
 
+// Unused. Kept here to maintain compatibility.
```

Comment:
If this is part of the documented public API surface and no longer does anything, then a release including this PR would be a breaking change (changed behaviour). In which case it shouldn't stick around as it'll act as a source of confusion. Same does for `serializers` beflow.

Reply:
That's a good point. This isn't documented anywhere. What do you think about just removing these constants and document that they were removed?

Reply:
SGTM, but maintainers would have the final call.

Reply:
How do I get a maintainer to look at this PR? :D

Reply:
@shilman ?